win32ss/user/ntuser/defwnd.c

```diff
@@ -414,7 +414,7 @@ UserPaintCaption(PWND pWnd, INT Flags)
          * RealUserDrawCaption in order to draw the classic caption when themes
          * are disabled but the themes service is enabled
          */
-         TRACE("UDCB Flags %08x\n", Flags);
+         TRACE("UDCB Flags %08x\n");
```

Comment:
This change is wrong, and makes me think that you created your PR from an outdated state.